### DISCUSSION TITLE
What is the cost of running dogehouse ?

### DISCUSSION THREAD

**Discussion by MohamedAliRashad** [benawad/dogehouse#2845] [Q&A] [answered]
I am interested to know how much money is needed to run dogehouse monthly ?

**Comment by parkerthegeniuschild** [comment]
Same here . . .

**Comment by chsaikumar445** [comment]
it depends on how many people are using it and things like that

**Comment by theotterlord** [comment, marked as answer]
This issue is more suited to be a discussion, but to answer the question, the only thing we have in terms of cost, is a segment at the end of [this video](https://youtu.be/oa84RnBsieo?t=405)
